frontend/src/components/atomic/team-member/TeamMember.tsx

```diff
@@ -12,10 +42,10 @@ export const TeamMember: React.FC<TeamMemberProps> = ({
   role,
 }: TeamMemberProps): JSX.Element => {
   return (
-    <div>
-      <img src={pictureRoute} alt="" />
-      <h1>{fullName}</h1>
-      <p>{role}</p>
-    </div>
+    <Container>
+      <Image src={pictureRoute} alt="" />
```

Comment:
```suggestion
      <Image src={pictureRoute} alt={fullName} />
```
De este modo mantenemos algo de usabilidad.